crates/katana/core/src/backend/mod.rs

```diff
@@ -171,7 +171,11 @@ impl Backend {
             receipts,
         )?;
 
-        info!(target: "backend", "⛏️ Block {block_number} mined with {tx_count} transactions");
+        info!(
+            target: "backend", "⛏️ Block mined with transactions",
```

Comment:
```suggestion
            target: "backend", "Block mined with transactions.",
```

Generally prefer to leave emojis out of logs so they are more easily machine readable. cc @kariy to confirm he agrees here.

Reply:
agree